libs/testdiff/golden.go

```diff
@@ -16,7 +17,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var OverwriteMode = os.Getenv("TESTS_OUTPUT") == "OVERWRITE"
+var OverwriteMode = false
+
+func init() {
+	flag.BoolVar(&OverwriteMode, "update", false, "Overwrite golden files")
+}
```

Comment:
This changes a global. Do you know for sure this package is not imported in the main CLI?

Alternatively, it can be a function to initialize it and call it from the acceptance test main function.

Reply:
It's not imported in the main CLI:

```
~/work/cli % GODEBUG=inittrace=1 ./cli 2>&1 | grep libs/t
init github.com/databricks/cli/libs/tags @7.2 ms, 0.075 ms clock, 76088 bytes, 402 allocs
init github.com/databricks/cli/libs/template @8.6 ms, 0.013 ms clock, 192 bytes, 4 allocs

~/work/cli % ./cli --version
Databricks CLI v0.0.0-dev+3be91f7da7e2
```